wdio.conf.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const localIP = require('ip');
+const path = require('path');
+
+const staticServerPort = 8080;
+
+const config = {
+  ...wdioConf.config,
+
+  baseUrl: `http://${localIP.address()}:${staticServerPort}`,
+  specs: [
+    path.join(__dirname, 'packages', '**', 'tests', 'specs', '**'),
+  ],
+  staticServerPort,
+  staticServerLog: false,
+  staticServerFolders: [
+    { mount: '/', path: path.join(__dirname, 'packages', 'terra-site', 'dist') },
+  ],
+
+  suites: {
+    badge: [
+      path.join(__dirname, 'packages', 'terra-badge', 'tests', 'specs', '**'),
```

Comment:
I think we can create a function that pulls all the packages and auto-generates suite names...